admin/js/editor.js

```diff
@@ -12,4 +13,23 @@ wp.domReady(() => {
     unregisterBlockStyle('planet4-blocks/covers', 'take-action');
     unregisterBlockStyle('planet4-blocks/covers', 'campaign');
   }
+
+  // Remove Button block native styles
+  unregisterBlockStyle('core/button', 'fill');
+  unregisterBlockStyle('core/button', 'outline');
+
+  // Add our custom Button block styles
+  const styles = [
+    {
+      name: 'secondary',
+      label: __('Secondary', 'planet4-blocks-backend'),
+      isDefault: true
+    },
+    {
+      name: 'cta',
+      label: __('Primary', 'planet4-blocks-backend')
```

Comment:
Mag agreed it made more sense to rename this style to `Primary` (it used to be `CTA`)